docs/source/release/v6.10.0/Framework/Algorithms/New_features/37141.rst

```diff
@@ -0,0 +1 @@
+- Re-implemtation of :ref:`LoadEventNexus <algm-LoadEventNexus>` when specifying the ``CompressTolerance``. This uses significantly less memory to create fewer events overall. However, the execution time of ``LoadEventNexus`` itself is generally longer; workflows that benefit from ``CompressEvents`` early in them generally run faster.
```

Comment:
suggest: "workflows that benefit from ``CompressEvents`` generally will run faster"